src/antares/service/api_services/run_api.py

```diff
@@ -0,0 +1,120 @@
+        url = f"{self._base_url}/launcher/run/{self.study_id}"
+        try:
+            if parameters is not None:
+                payload = parameters.to_api()
+                response = self._wrapper.post(url, json=payload)
+            else:
+                response = self._wrapper.post(url)
+            job_id = response.json()["job_id"]
+            return self._get_job_from_id(job_id)
+        except APIError as e:
+            raise AntaresSimulationRunningError(self.study_id, e.message) from e
+
+    def _get_job_from_id(self, job_id: str) -> Job:
+        url = f"{self._base_url}/launcher/jobs/{job_id}"
+        response = self._wrapper.get(url)
+        job_info = response.json()
+        status = JobStatus.from_str(job_info["status"])
+        launcher_params = json.loads(job_info["launcher_params"])
+        output_id = job_info["output_id"] if status == JobStatus.SUCCESS else None
+        return Job(job_id=job_id, status=status, unzip_output=launcher_params["auto_unzip"], output_id=output_id)
+
+    def wait_job_completion(self, job: Job, time_out: int) -> None:
+        start_time = time.time()
+        repeat_interval = 5
+        if job.status == JobStatus.SUCCESS:
+            self._update_job(job)
+
+        while job.status in (JobStatus.RUNNING, JobStatus.PENDING):
+            if time.time() - start_time > time_out:
+                raise SimulationTimeOutError(job.job_id, time_out)
+            time.sleep(repeat_interval)
+            self._update_job(job)
+
+        if job.status == JobStatus.FAILED or not job.output_id:
+            raise SimulationFailedError(self.study_id)
+
+        if job.unzip_output:
+            self._wait_unzip_output(self.study_id, job.output_id, time_out)
+
+        return None
+
+    def _update_job(self, job: Job) -> None:
+        updated_job = self._get_job_from_id(job.job_id)
+        job.status = updated_job.status
+        job.output_id = updated_job.output_id
+
+    def _wait_unzip_output(self, ref_id: str, job_output_id: str, time_out: int) -> None:
+        url = f"{self._base_url}/tasks"
+        repeat_interval = 2
+        payload = {"type": ["UNARCHIVE"], "ref_id": ref_id}
+        try:
+            response = self._wrapper.post(url, json=payload)
+            tasks = response.json()
+            task_id = self._get_unarchiving_task_id(job_output_id, tasks)
+            self._wait_task_completion(task_id, repeat_interval, time_out)
+        except APIError as e:
+            raise AntaresSimulationUnzipError(self.study_id, e.message) from e
+
+    def _get_unarchiving_task_id(self, job_output_id: str, tasks: list[dict[str, Any]]) -> str:
+        for task in tasks:
+            task_name = task["name"]
+            output_id = task_name.split("/")[-1].split(" ")[0]
+            if output_id == job_output_id:
+                return task["id"]
+        raise AntaresSimulationUnzipError(self.study_id, "Could not find task for unarchiving job")
```

Comment:
We should add the job id at the end of the log. It could help the user and also us debugging